internal/output/output.go

```diff
@@ -288,9 +288,16 @@ func outputResource(r *schema.Resource) Resource {
 		subresources = append(subresources, outputResource(s))
 	}
 
+	metadata := make(map[string]string)
+	if r.InfracostMetadata != nil {
+		for k, v := range r.InfracostMetadata {
+			metadata["infracost_"+k] = v.String()
```

Comment:
If I'm understanding this correctly this is just to store data in the "plan JSON-style" output we then pass the the plan JSON parser? What do we see being in there longer term? I'm assuming things like line numbers as well.

Reply:
Also do we need to prefix these with `infracost_`? Since this goes into the Infracost JSON format then it seems redundant to have that prefix.

Reply:
Yeah I think line numbers are probably the next thing we add. Re the Infracost prefix, yeah I agree it's not needed. Not really sure why it was there.